e2e/client/playwright/authoring.spec.ts

```diff
@@ -38,3 +38,28 @@ test('can apply "populate abstract" macro', async ({page}) => {
         page.locator(s('authoring', 'authoring-field=abstract')).getByRole('textbox'),
     ).toHaveText('test sport story body');
 });
+
+test('can embed if url is valid', async ({page}) => {
```

Comment:
I'd call it "can add embeds". If you want to test invalid urls, I would add another test "editor3 doesn't crash when attempting to add invalid embeds"

Also, it's better to move this test to `editor3.spec.ts` so we separate editor stuff from the rest of authoring tests.